My Project/Form_Main.vb

```diff
@@ -3028,8 +3032,17 @@ Public Class Form_Main
         Dim UC As New UtilsCommon
 
         Dim columnIndex As Integer = hitinfo.Item.SubItems.IndexOf(hitinfo.SubItem)
-        Dim PropertySet As String = TemplatePropertyDict(hitinfo.Item.ListView.Columns.Item(columnIndex).Text)("PropertySet")
-        Dim PropertyNameEnglish = TemplatePropertyDict(hitinfo.Item.ListView.Columns.Item(columnIndex).Text)("EnglishName")
+        Dim PropertySet As String = ""
+        Dim PropertyNameEnglish = ""
+
+        Try
```

Comment:
@rmcanany This is a temporary path, I have added it only here cause there is duplicate code that needs to be consolidated. Just track the issue by now we can do a proper fix when consolidation is done.